### PR TITLE
feat(terraform): add CKV NCP rules about ncloud access control group rule

### PR DESCRIPTION
**Pull request by pj991207**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description
Add terraform scan rules about provider Naver Cloud Platform.
CKV_NCP_26 is associated with access control group has access control group rule

### Description
https://registry.terraform.io/providers/NaverCloudPlatform/ncloud/latest/docs/resources/lb_listener

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [ ] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by nimrodkor — APPROVED** [review]

File: checkov/terraform/checks/graph_checks/ncp/AccessControlGroupRuleDefine.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+metadata:
+  id: "CKV_NCP_26"
+  name: "Ensure Access Control Group has Access Control Group Rule"
```
Comment (gruebel) [inline checkov/terraform/checks/graph_checks/ncp/AccessControlGroupRuleDefine.yaml:3]:
```suggestion
  name: "Ensure Access Control Group has Access Control Group Rule attached"
```

File: checkov/terraform/checks/graph_checks/ncp/AccessControlGroupRuleDefine.yaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  category: "GENERAL_SECURITY"
```
Comment (gruebel) [inline checkov/terraform/checks/graph_checks/ncp/AccessControlGroupRuleDefine.yaml:4]:
```suggestion
  category: "NETWORKING"
```

**Review by gruebel — COMMENTED** [review]
nice work, added to small comments

**Review by gruebel — APPROVED** [review]
🍻

**Pull request merged**